src/EA.Weee.Web/Areas/Admin/Controllers/ProducerSubmissionController.cs

```diff
@@ -137,7 +138,7 @@ public async Task<ActionResult> AddPaymentDetails()
 
         [AdminSmallProducerSubmissionContext]
         [HttpGet]
```

Comment:
@stevelee-civica  Can we also have the [AuthorizeInternalClaims(Claims.InternalAdmin)] on the RemoveSubmission GET and Remove GET action and also unit test to check these attributes have this attribute.

Reply:
added unit tests